src/services/codefixes/interfaceFixes.ts

```diff
@@ -0,0 +1,268 @@
+/* @internal */
+namespace ts.codefix {
+    registerCodeFix({
+        errorCodes: [Diagnostics.Type_0_is_not_assignable_to_type_1.code],

```

Comment:
this code fix is not nearly close to being complete. please remove this from this patch and added by itself later on. i am not sure we should do this either. there are issues that are not fixable. and we are not detecting them and handling them correctly.